src/dhcp6relay/src/relay.cpp

```diff
@@ -450,10 +405,8 @@ void callback(evutil_socket_t fd, short event, void *arg) {
 
     auto dhcp_header = parse_dhcpv6_hdr(current_position);
 
-    if ((ntohs(udp_header->dest)) == 547 ) {
-        if(dhcp_header->msg_type == SOLICIT || REQUEST || CONFIRM || RENEW || REBIND || RELEASE || DECLINE) {
-            relay_client(config->local_sock, current_position, ntohs(udp_header->len) - sizeof(udphdr), ip_header, ether_header, config);
-        }    
+    if (((ntohs(udp_header->dest)) == RELAY_PORT) && (dhcp_header->msg_type == SOLICIT || REQUEST || CONFIRM || RENEW || REBIND || RELEASE || DECLINE)) {
```

Comment:
This condition will always be true:
`(dhcp_header->msg_type == SOLICIT || REQUEST || CONFIRM || RENEW || REBIND || RELEASE || DECLINE)`